Src/FluentAssertions/Primitives/DateTimeOffsetRangeAssertions.cs

```diff
@@ -132,5 +132,5 @@ private static string PositionRelativeToTarget(DateTimeOffset actual, DateTimeOf
 
     /// <inheritdoc/>
     public override bool Equals(object obj) =>
-        throw new NotSupportedException("Calling Equals on Assertion classes is not supported.");
+        throw new NotSupportedException("Equals is not part of Fluent Assertions. Did you mean Before() or After() instead?");
```

Comment:
🤔 I would also expect `Be` here

Reply:
This is the range assertion class.  It's exceedingly unlikely this will ever get hit, but I think this is right.  Take a look at the unit test I made for it.

https://github.com/fluentassertions/fluentassertions/pull/2006/files#diff-b7687164262012a0106e62fac13a776ad7481b1efe93346fe179efa32a096caeR2300

There wasn't a `DateTimeOffsetRangeAssertionSpec.cs` so I just put it in the `DateTimeOffsetAssertionSpec.cs`. For the non-range one, this PR does suggest `Be()`.  I can create a new `DateTimeOffsetRangeAssertionSpec.cs` file if you'd like.